force-app/main/default/lwc/scorePD1Calculator/scorePD1Calculator.html

```diff
@@ -0,0 +1,47 @@
+<template>
+    <lightning-card title="PD1 Calculator">
+        <!-- <div  title="PD1 Calculator"></div> -->
+            <lightning-button variant="brand" label="Calculate" title="Calculate" slot="actions" onclick={handleClick}></lightning-button> 
+                <div class="slds-var-p-horizontal_small">    
+                    <lightning-input class="slds-container_center" type="number" name="devFundamentals" label="Developer Fundamentals :23%" value ={devFundamentalScore} onchange={handleChange}></lightning-input>
```

Comment:
```suggestion
                    <lightning-input
                        class="slds-container_center"
                        type="number"
                        name="devFundamentals"
                        label="Developer Fundamentals :23%"
                        value ={devFundamentalScore}
                        onchange={handleChange}></lightning-input>
```
When you start to add a lot of modifiers in your HTML, think about stacking them up like this. Yes the code will become a little longer, but it's easier to read them quickly and tell what value each item is set too.